src/zkevm_specs/evm/table.py

```diff
@@ -38,6 +40,62 @@ class FixedTableTag(IntEnum):
     StackOverflow = auto()  # opcode, stack_pointer, 0
     StackUnderflow = auto()  # opcode, stack_pointer, 0
 
+    def table_assignments(self) -> Sequence[Array4]:
+        if self == FixedTableTag.Range16:
+            return [(self, i, 0, 0) for i in range(16)]
+        elif self == FixedTableTag.Range32:
+            return [(self, i, 0, 0) for i in range(32)]
+        elif self == FixedTableTag.Range64:
+            return [(self, i, 0, 0) for i in range(64)]
+        elif self == FixedTableTag.Range256:
+            return [(self, i, 0, 0) for i in range(256)]
+        elif self == FixedTableTag.Range512:
+            return [(self, i, 0, 0) for i in range(512)]
+        elif self == FixedTableTag.Range1024:
+            return [(self, i, 0, 0) for i in range(1024)]
+        elif self == FixedTableTag.SignByte:
+            return [(self, i, (i >> 7) * 0xFF, 0) for i in range(256)]
```

Comment:
If we want to just retain the sign, shouldn't we AND against `0xFF`?
```suggestion
            return [(self, i, (i >> 7) & 0xFF, 0) for i in range(256)]
```

Reply:
IIRC this table contains a byte with all bits set if the input byte is negative (in two's complement), that's why it's multiplied by `0xFF`.  Maybe a comment on the table generation code could be added to clarify this?

Reply:
Hmmm! Makes sense. I missed it. It's clear then! 
Thanks!!